### PR TITLE
Xontrib readable traceback

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
This is `noarch: python` and so is ready for review

**Comment by scopatz** [comment]
ping @conda-forge/staged-recipes

File: recipes/xontrib-readable-traceback/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - colorama
```
Comment (chrisburr) [inline recipes/xontrib-readable-traceback/meta.yaml]: Looking at the setup.py I don't think you need these?
```suggestion
```

File: recipes/xontrib-readable-traceback/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - backtrace
```
Comment (chrisburr) [inline recipes/xontrib-readable-traceback/meta.yaml]:
```suggestion
```

**Pull request merged**